src/Employer/Employer.Web/Controllers/VacancyPreviewController.cs

```diff
@@ -89,6 +89,14 @@ public async Task<IActionResult> Review(SubmitReviewModel m)
         [HttpPost("preview", Name = RouteNames.Preview_Submit_Post)]
```

Comment:
I know this wasn't part of the original change, however we should change the URL it goes to on POST, as if you now reload the page at this URL you'll see the old preview page

Reply:
`"advert-preview"` is the newer one i believe

Reply:
Amended in latest.